src/components/dataset/DatasetAccess.jsx

```diff
@@ -20,16 +31,25 @@ function DatasetAccess(props) {
       dispatch(removeDatasetPolicyMember(dataset.id, removableEmail, role));
     };
   };
-  const { horizontal, policies, userRoles } = props;
+  const { horizontal, policies, userRoles, showHelp, classes } = props;
   const stewards = getRoleMembersFromPolicies(policies, DatasetRoles.STEWARD);
   const custodians = getRoleMembersFromPolicies(policies, DatasetRoles.CUSTODIAN);
   const snapshotCreators = getRoleMembersFromPolicies(policies, DatasetRoles.SNAPSHOT_CREATOR);
 
-  const canManageUsers = userRoles.includes(DatasetRoles.STEWARD);
+  const canManageUsers =
+    userRoles.includes(DatasetRoles.STEWARD) || userRoles.includes(DatasetRoles.CUSTODIAN);
```

Comment:
@nmalfroy - the same note about Custodians not being able to create Stewards applies here too, right?

Reply:
I think I've addressed it here already for datasets. for the other one, the only people who can manage users for snapshots are stewards - the Custodian role doesnt exist at all, so I didnt touch that logic!